pkg/utils/utils.go

```diff
@@ -196,13 +197,30 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 		if err != nil {
 			glog.Warningf("configSriovDevice(): unable to parse VFs for device %+v %q", iface.PciAddress, err)
 		}
+		ifaceLink, err := netlink.LinkByName(iface.Name)
```

Comment:
@Mmduh-483  
Would you please create a separate function for changing admin MAC address and put it right after `setSriovNumVfs` is called? With this, it will only be called once when VFs are provisioned and it's guaranteed that all the VFs will be kernel netdevice interfaces (rather than bind to vfio-pci).

Reply:
Changed